select.go

```diff
@@ -216,14 +219,16 @@ func (s *Select) RunCursorAt(cursorPos, scroll int) (int, string, error) {
 }
 
 func (s *Select) innerRun(cursorPos, scroll int, top rune) (int, string, error) {
-	stdin := readline.NewCancelableStdin(os.Stdin)
-	c := &readline.Config{}
+	c := &readline.Config{
+		Stdin:  s.Stdin,
+		Stdout: s.Stdout,
+	}
 	err := c.Init()
 	if err != nil {
 		return 0, "", err
 	}
 
-	c.Stdin = stdin
+	c.Stdin = readline.NewCancelableStdin(c.Stdin)
```

Comment:
While thinking about it, this line is likely superfluous now (and might even prevent people from using the a "non-cancelable" stdin).